src/compiler/binder.ts

```diff
@@ -531,6 +537,48 @@ namespace ts {
             typeLiteralSymbol.members = { [symbol.name]: symbol };
         }
 
+        function bindObjectLiteralExpression(node: ObjectLiteralExpression) {
+            if (inStrictMode) {
+                let seen: Map<number> = {};
+                const Property = 1;

```

Comment:
It would be nice to comment that you picked 1 & 2 for the sake of truthiness.

Also, consider using a local const enum.